K8s/k8s-mongo.sh

```diff
@@ -1,15 +1,16 @@
+#Create soda-multi-cloud namespace
 kubectl create namespace soda-multi-cloud
 
-kubectl apply -f mongo-pv-0.yaml -n soda-multi-cloud
-kubectl apply -f mongo-pv-1.yaml -n soda-multi-cloud
-kubectl apply -f mongo-pv-2.yaml -n soda-multi-cloud
+kubectl apply -f mongo-pv-0.yaml 
+kubectl apply -f mongo-pv-1.yaml 
+kubectl apply -f mongo-pv-2.yaml 
  
-kubectl apply -f mongo-pvc-0.yaml -n soda-multi-cloud
-kubectl apply -f mongo-pvc-1.yaml -n soda-multi-cloud
-kubectl apply -f mongo-pvc-2.yaml -n soda-multi-cloud
+kubectl apply -f mongo-pvc-0.yaml 
+kubectl apply -f mongo-pvc-1.yaml 
+kubectl apply -f mongo-pvc-2.yaml 
 
-kubectl apply -f mongo-service-0.yaml -n soda-multi-cloud
-kubectl apply -f mongo-service-1.yaml -n soda-multi-cloud
-kubectl apply -f mongo-service-2.yaml -n soda-multi-cloud
+kubectl apply -f mongo-service-0.yaml 
+kubectl apply -f mongo-service-1.yaml 
+kubectl apply -f mongo-service-2.yaml 
 
```

Comment:
can we merge k8s-mongo.sh and k8s-deploy.sh files

Reply:
No

Reply:
Will do once stable..